src/libraries/Microsoft.Extensions.Configuration.Xml/tests/ConfigurationProviderXmlTest.cs

```diff
@@ -20,6 +20,16 @@ public override void Combine_after_other_provider()
             // Disabled test due to XML handling of empty section.
         }
 
+        public override void Load_from_single_provider_with_duplicates_throws()
```

Comment:
Should we be doing the same thing as below instead?
https://github.com/dotnet/runtime/blob/8fcb543625324c81f934e67a15077ad903dd3c9f/src/libraries/Microsoft.Extensions.Configuration.Json/tests/ConfigurationProviderJsonTest.cs#L13-L19

and 

https://github.com/dotnet/runtime/blob/8fcb543625324c81f934e67a15077ad903dd3c9f/src/libraries/Microsoft.Extensions.Configuration.CommandLine/tests/ConfigurationProviderCommandLineTest.cs#L38-L49

Reply:
Well the behavior with duplicate keys is already covered by other tests. These tests in the JSON provider simply test if the provider does not crash on duplicate keys, while my added tests also verify if the actual output is correct.

But when in doubt, it's always better to be consistent. I've followed the JSON example and implemented these tests the same for the XML provider.